environments/staging/inventory.ini.j2

```diff
@@ -141,6 +141,21 @@ es8
 [shared_dir_host:children]
 redis2
 
+[shared_dir_host:vars]
+efs_staging_mount={{ aws_resources['efs-staging-mount-dns'] }}:/
+cchq_uid=nobody
+cchq_gid=nfs
+
+[shared_efs_client_host:children]
+celery
+django_manage
+proxy
+pillowtop
+webworkers
+
+[shared_efs_client_host:vars]
+efs_staging_mount={{ aws_resources['efs-staging-mount-dns'] }}:/
```

Comment:
Might there be a way to do all of this hijacking the existing code in the `shared_dir` role? I'm imagining (1) make [generated variables](https://github.com/dimagi/commcare-cloud/blob/52f78b7e16c6b4a871755cc41344edd4ee5f125a/src/commcare_cloud/environment/main.py#L339) include "shared_mount_address` set to `aws_resources['efs-staging-mount-dns']` and  `shared_mount_opts: "_netdev,nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2,noresvport"` like you have below (2) remove `shared_dir_host` from the inventory entirely. That way you don't have to create this new `shared_efs_client_host` group.

Reply:
I have updated below files with **_shared_mount_opts_** variable & value.
- environments/staging/meta.yml
- src/commcare_cloud/environment/main.py
- src/commcare_cloud/environment/schemas/meta.py

I had tried to use **shared_mount_address**  variable & value as {{ aws_resources['efs-staging-mount-dns'] }}:/  but I could not executed it due to "undefined variable. The error was: 'aws_resources' is undefined" error. So, I have mentioned shared_mount_address ( efs_shared_mount_endpoint ) varibale in public.yml file.